src/dataset/mod.rs

```diff
@@ -20,14 +21,17 @@ pub struct Dataset {
     pub tags: Vec<String>,
     pub source_url: String,
     pub resources: Vec<Resource>,
+    pub issued: Option<Date>,
 }
 
 #[derive(Debug, Deserialize, Serialize)]
 struct OldDataset {
     pub title: String,
     pub description: String,
     pub license: License,
+    pub tags: Vec<String>,
```

Comment:
`tags` and `resources` are not yet present on the production, please remove those. (When we do add them, `tags` would need to migrated, e.g. `tags: old_val.tags`.